packages/frontend/src/components/select_sports.tsx

```diff
@@ -0,0 +1,47 @@
+import type { Sport } from "@esp-group-one/types";
+import { useState } from "react";
+import type { ChangeEvent } from "react";
+import { sportToColour } from "../../util/sport";
+
+interface SportListProps {
+  sports: Sport[];
+}
+
+export function SelectSport({
+  sports: sportsOptions,
+  currentSport,
+}: SportListProps & { currentSport: Sport }) {
+  const [sport, setSport] = useState(currentSport.toString());
+
+  const handleChange = (e: ChangeEvent<HTMLSelectElement>) => {
+    setSport(e.target.value);
+  };
+
+  return (
+    <div>
+      <select
+        className={`text-white ${sportToColour(
+          sport,
+        )} hover:bg-blue-800 focus:ring-4 focus:outline-none focus:ring-blue-300 font-body rounded-lg text-lg px-5 py-2.5 text-center inline-flex items-center w-full`}
+        value={sport}
+        id="dropdown"
+        onChange={(e) => {
+          handleChange(e);
+        }}
```

Comment:
This is an unnecessary wrapper, you should be able to just pass in  `handleChange`